app/views/layouts/darkswarm.html.haml

```diff
@@ -2,7 +2,9 @@
   %head
     %meta{charset: 'utf-8'}/
     %meta{name: 'viewport', content: "width=device-width,initial-scale=1.0"}/
-
+    %meta{:content => yield(:title).empty? ? "#{t(:title)}" : yield(:title), :property => "og:title"}/
+    %meta{:content => yield(:description).empty? ? "#{t(:og_description)}" : yield(:description), :property => "og:description"}/
```

Comment:
I would suggest to call this i18n key `og_default_description`, so that it is clear that this is something that's not always used, but only when a description is missing.